internal/cmd/keepassxctemplatefuncs.go

```diff
@@ -100,6 +113,18 @@ func (c *Config) keepassxcTemplateFunc(entry string) map[string]string {
 }
 
 func (c *Config) keepassxcAttributeTemplateFunc(entry, attribute string) string {
+	if c.Keepassxc.Mode == keepassxcModeBuiltin {
+		// builtin stores attributes in cache
+		if data, ok := c.Keepassxc.cache[entry]; ok {
```

Comment:
As the library handles default values the same as attributes, both functions use the same cache in builtin mode.
I don't think that i makes sense to keep separate caches in this case.